test/boost/cdc_test.cc

```diff
@@ -209,7 +219,7 @@ SEASTAR_THREAD_TEST_CASE(test_detecting_conflict_of_cdc_log_table_with_existing_
 
         // Conflict on CREATE INDEX which enables cdc log for vector search
         BOOST_REQUIRE_THROW(e.execute_cql("CREATE INDEX ON ks.tbl (b) USING 'vector_index'").get(), exceptions::invalid_request_exception);
-    }).get();
+    }, enable_tablets()).get();
```

Comment:
The commit message says that for some not-very-clear-to-me reason (fashion?), we never ever want to test vector search with vnodes any more. I guess that's fine, since vector search as a brand-new feature and nobody will be using it on old existing tables.

BUT, you are changing here CDC tests. CDC is still live on upgraded installations with pre-existing vnode-tables. We do need to make sure CDC continues to work on vnodes - we can't claim we don't need to test it on vnodes any more! So I think these changes to cdc_test.cc are wrong.

By the way, I'm not sure I understand what happens without this new enable_tablets() call. When this test runs, what will it use, tablets or vnodes? Or do we have two runs on CI, one with tablets and one with vnodes?